CMakeLists.txt

```diff
@@ -559,6 +559,7 @@ if((ENABLE_LOG) AND (CMAKE_SYSTEM_NAME STREQUAL "Linux"))
 
     set (install-3rd_party_logger_var "install-3rd_party_logger")
   endif()
+  set(LOG4CXX_LOGGER ON)
```

Comment:
@anosach-luxoft, 
I agree that logs is helpful for testing result analisys, but it is not correct to using different states inside and out of test.
As an example - we disable in some component - it builds without logger macros, test are treating the component header with enabled logs as the same, but binary they are not compatible.